content/en/integrations/guide/storage-monitoring-setup.md

```diff
@@ -85,6 +85,9 @@ As an alternative, you can set up S3 inventory manually or with Terraform and en
               }
         ```
 
+        **Storage Management relies on AWS Integration and Resource Collection** to bring together bucket details. Add all S3 permissions from the [Integration IAM policy and Resource Collection IAM policy][502] in addition to the permissions above.
```

Comment:
I noticed that we linked to the same page in the paragraph above the code sample, and the information in the two paragraphs seems similar. Can we combine them into one?